caravel/templates/caravel/explore.html

```diff
@@ -147,7 +147,7 @@
                 <span class="legend_label">{{ _("Filters") }}</span>
                 <i  class="fa fa-info-circle" data-toggle="tooltip"
                     data-placement="bottom"
-                    title="{{_("Filters are defined using comma delimited strings as in 'US,FR,Other'")}}"></i>
+                    title="{{_("Filters are defined using comma delimited strings as in 'US,FR,Other'")}}. Leave the value field empty to filter empty strings or nulls"></i>
```

Comment:
`_('Translate me')` is used for translation, let's try to encapsulate most strings!